src/components/PlayerSubmissionForm.js

```diff
@@ -27,11 +57,15 @@ const PlayerSubmissionForm = () => {
       </form>
     </div>
   );
+} else {
+  return null
+}
 }
 
 PlayerSubmissionForm.propTypes = {
   index: PropTypes.number.isRequired,
   sendSubmission: PropTypes.func.isRequired,
+  isSubmitted: PropTypes.bool.isRequired,
```

Comment:
Just a note, that because you've added this prop the tests are giving a warning because the tests aren't using.  If you add that prop to the tests, the warning will stop.